### ISSUE TITLE
How mt5 checkpoint is selected on TydiQA dataset

### ISSUE THREAD

**Issue by dorost1234** [google-research/multilingual-t5#69]
Hi
This dataset only have train/validation set, I assume the reported numbers are on validation set, in that case, could you clarify please how you select the best checkpoint ? on which dataset validation is done? thank you

**Comment by lintingxue** [comment]
hi,
We report the numbers on the validation set and the best checkpoint is
chosen based on the average F1 across languages.

Tydiqa dev is defined in tasks. py
<https://github.com/google-research/multilingual-t5/blob/a694d1af8d52de5c46b649cb1b9e3b0ba8405a88/multilingual_t5/tasks.py#L283>.
We use the standard Tydiqa validation in XTREME
<https://github.com/google-research/xtreme>. Dataset is also available in
TFDS <https://www.tensorflow.org/datasets/catalog/tydi_qa>

**Comment by dorost1234** [comment]
Hi
I do not really understand this, could you please explain properly?
You mentioned this is zero-shot, then this was not zero shot? since you
choose a checkpoint on the validation set of the languages?
Second you choose the checkpoint based on validation performance? then also
report on the validation set? thanks

**Comment by dorost1234** [comment]
Have you fine-tuned then on the test set? this is very wrong then
